04-ed-cognitions.Rmd

```diff
@@ -5,7 +5,7 @@ Eating disorder cognitions were assessed in-depth at age 14 years, with some add
 ```{r}
 #insert table here with constructs, ages, and checkmarks
 ```
-
```

Comment:
I don't think you should have deleted this

Reply:
i think its fine

Reply:
ok